features/features.go

```diff
@@ -508,4 +508,11 @@ var (
 						contactPerson("rvanderp3").
 						productScope(ocpSpecific).
 						mustRegister()
+
+	FeatureGateChunkSizeMiB = newFeatureGate("ChunkSizeMiB").
+				reportProblemsToJiraComponent("Image Registry").
+				contactPerson("flavianmissi").
+				productScope(ocpSpecific).
+				enableIn(DevPreviewNoUpgrade, TechPreviewNoUpgrade).
```

Comment:
This isn't right and causes build failures
```suggestion
				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```